build/status/MV93cXBkYXRhL291dC9kYXRhL01haW5lX25pdHJhdGVfMDAxX3NpdGVpbmZvLmZlYXRoZXIuaW5k.yml

```diff
@@ -0,0 +1,11 @@
+version: 0.3.0
+name: 1_wqpdata/out/data/Maine_nitrate_001_siteinfo.feather.ind
```

Comment:
could also skip committing these build/status files for these small siteinfo files

Reply:
That sounds good; it would be great to have these small files for now while I'm still working things out with Melissa.  I need to modify some of the crosswalk relationships based on feedback from Gretchen, so I'm ready to start reprocessing as soon as you guys are happy with the data pulls and .ind/siteinfo files, and can load them up on the google drive.